src/dataExplorer/context/results/childResults.tsx

```diff
@@ -132,6 +131,10 @@ export const ChildResultsProvider: FC = ({children}) => {
         language: resource?.language ?? LanguageType.SQL,
         bucket: selection.bucket,
         sqlQueryModifiers,
+        // keep SQL using the Flux endpoint so that
+        // UI can keep reusing the Giraffe plots
+        // https://github.com/influxdata/idpe/issues/17986#issuecomment-1703346936
+        useFluxEndpoint: true,
```

Comment:
When user select SQL as the query language, UI makes a Flux query to get the data for graphs